src/components/content-tab-import-export.tsx

```diff
@@ -86,20 +99,170 @@ export const ExportSite = ( {
+		>
+			{ children }
+		</Button>
+	) : (
+		<div className="w-full">{ children }</div>
+	);
+
+const ImportSite = ( props: { selectedSite: SiteDetails } ) => {
+	const { __ } = useI18n();
+	const { importFile, updateSite, startServer, loadingServer } = useSiteDetails();
+	const importConfirmation = useConfirmationDialog( {
+		message: sprintf( __( 'Overwrite %s?' ), props.selectedSite.name ),
+		checkboxLabel: __( "Don't ask again" ),
+		detail: __( 'Importing a backup will replace the existing files and database for your site.' ),
+		confirmButtonLabel: __( 'Import' ),
+		localStorageKey: 'dontShowImportConfirmation',
+	} );
+
+	const { dropRef, isDraggingOver } = useDragAndDropFile< HTMLDivElement >( {
+		onFileDrop: ( file: File ) => {
+			importConfirmation( () => importFile( file, props.selectedSite ) );
+		},
+	} );
+	const inputFileRef = useRef< HTMLInputElement >( null );
+	const openFileSelector = async () => {
+		inputFileRef.current?.click();
+	};
+	const onFileSelected = async ( e: React.ChangeEvent< HTMLInputElement > ) => {
+		const file = e?.target?.files?.[ 0 ];
+		if ( ! file ) {
+			return;
+		}
+		importConfirmation( async () => {
+			await importFile( file, props.selectedSite );
+			clearImportFileInput();
+		} );
+	};
+	const openSite = async () => {
+		if ( ! props.selectedSite.running ) {
+			speak( __( 'Starting the server before opening the site link' ) );
+			await startServer( props.selectedSite.id );
+		}
+		getIpcApi().openSiteURL( props.selectedSite.id );
+	};
+	const clearImportFileInput = () => {
+		if ( inputFileRef.current ) {
+			inputFileRef.current.value = '';
+		}
+	};
+	const clearImportState = () => {
+		updateSite( { ...props.selectedSite, importState: ImportState.Initial } );
+		clearImportFileInput();
+	};
+
+	const startLoadingCursorClassName =
+		loadingServer[ props.selectedSite.id ] && 'animate-pulse duration-100 cursor-wait';
+
+	const isImporting = props.selectedSite.importState === ImportState.Importing && ! isDraggingOver;
+	const isImported = props.selectedSite.importState === ImportState.Imported && ! isDraggingOver;
+	const isInitial = ! isImporting && ! isImported;
+	return (
+		<div className={ cx( 'flex flex-col w-full', startLoadingCursorClassName ) }>
+			<div className="a8c-subtitle-small mb-1">{ __( 'Import' ) }</div>
+			<div className="text-a8c-gray-70 a8c-body mb-4">
+				{ createInterpolateElement(
+					__( 'Import a Jetpack backup or a .sql database file. <button>Learn more</button>' ),
+					{
+						button: (
+							<Button variant="link" onClick={ () => getIpcApi().openURL( STUDIO_DOCS_URL ) } />
+						),
+					}
+				) }
+			</div>
+			<div ref={ dropRef } className="w-full">
+				<InitialImportButton isInitial={ isInitial } openFileSelector={ openFileSelector }>
+					<div
+						className={ cx(
+							'h-48 w-full rounded-sm border border-zinc-300 flex-col justify-center items-center inline-flex',
+							isDraggingOver && 'border-a8c-blueberry bg-a8c-gray-0'
```

Comment:
Following the above comment, we shouldn't probably display the border while dragging to avoid confusion about the behavior.

Reply:
Changed https://github.com/Automattic/studio/pull/369/commits/9153649af06c7b98ae6fe0041d50e4d6798dc051#diff-fd133fa5ff9b8ef225c2775f90a5122225974fff6288e81b6fd518ef9bbb2e0bR212